packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/__brick__/lib/feature_onboarding/blocs/onboarding_phone_bloc_extensions.dart

```diff
@@ -0,0 +1,16 @@
+part of 'onboarding_phone_bloc.dart';
+
+extension OnboardingBlocExtensions on OnboardingPhoneBloc {
+  /// Validates the phone number
+  bool _validatePhoneNumber(String? countryCode, String phoneNumber) {
+    if (countryCode == null || countryCode.isEmpty) {
+      throw InvalidCountryCodeError();
+    } else if (phoneNumber.isEmpty) {
+      throw InvalidPhoneNumberError();
+    } else if (phoneNumber.length < 8) {
+      throw PhoneNumberTooShortError();
```

Comment:
Can we use the FieldErrorModel instead of creating separate models for each case ?

More details can be found here: https://github.com/Prime-Holding/rx_bloc/blob/master/examples/todoapp/lib/feature_todo_management/services/todo_validator_service.dart

Reply:
Done.